swan/values.yaml

```diff
@@ -97,6 +97,7 @@ jupyterhub:
       kubectl.kubernetes.io/default-container: notebook
     extraEnv:
       SWAN_DISABLE_NOTIFICATIONS: "true"
+      GALLERY_URL: "https://swan-gallery.web.cern.ch"
```

Comment:
The fact that you need to set the variable here means that, in our current code, we checked if the variable existed in multiple places (e.g. [1](https://github.com/swan-cern/jupyterhub-image/blob/master/jupyterhub.d/jupyterhub_config/kubespawner.py#L26), [2](https://github.com/swan-cern/systemuser-image/blob/master/systemuser.sh#L271) and [3](https://github.com/swan-cern/jupyter-extensions/blob/master/SwanContents/swancontents/templates/tree.html#L23)) but it was actually never set? For example in [3](https://github.com/swan-cern/jupyter-extensions/blob/master/SwanContents/swancontents/templates/tree.html#L23) the default was being taken instead?

Reply:
Can you clarify what you mean, @etejedor ? I'm not understanding.

Reply:
Also, AFAIK, that jupyterhub config file is no longer used.. It's a leftover from other times

Reply:
`GALLERY_URL` is defined here in this commit, but in the current master we have code that checks for this variable already (see links I provided). This means the variable was never set in k8s, and we just relied on defaults (see 3).

Reply:
Yes, ofc we relied in the defaults. This option is for ScienceBox (this was used in the past for Up2U, as we had a special gallery there). And by checking you mean "setting the config if a value is defined"? If so, yes. 
Also, as I said, the code from first link you sent is not used anywhere (this is smth that has been cleaned in Riccardo PR that is waiting for the changes for the Spark stuff).